client/test/contact_mapper.spec.js

```diff
@@ -0,0 +1,173 @@
+
+'use strict';
+const assert = require('chai').assert;
+const Immutable = require('immutable');
+
+const contactMapper = require('../app/libs/mappers/contact')
+
+const fixtures = {
```

Comment:
I think that we should move this into `test/fixtures/contact.js` because contact should also be tested into messages. cc @aenario, @m4dz